### PR TITLE
[VDA-StorageNode-Registry] 1.3.0

### LINKED ISSUE #135: [vda-storage-node] Disable withdrawals
Allow the contract owner to enable / disable staked VDA token withdrawals

### PR DESCRIPTION
**Pull request by ITStar10**
- Added enable/disable withdrawal feature
- Added test code for above feature

### CODE REVIEW

File: VDA-StorageNode-Registry/contracts/facets/VDAStorageNodeFacet.sol [hunk 1]
```diff
@@ -62,6 +63,27 @@ contract VDAStorageNodeFacet is IStorageNode {
     emit UpdateStakingRequired(isRequired);
   }
 
+  /**
+    * @dev see { IStorageNode }
+    */
+  function isWithdrawlEnabled() external view virtual override returns(bool) {
```
Comment (tahpot) [inline VDA-StorageNode-Registry/contracts/facets/VDAStorageNodeFacet.sol]: Typo. Should be `Withdrawal`.

File: VDA-StorageNode-Registry/ChangeLog.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+2024-01-18 (V1.3.0)
```
Comment (tahpot) [inline VDA-StorageNode-Registry/ChangeLog.md]: This code hasn't been officially released, so shouldn't have a version number yet.

You can just track all the changes without any version numbers or dates until we do an official release.

File: VDA-StorageNode-Registry/contracts/facets/VDAStorageNodeFacet.sol [hunk 3]
```diff
@@ -62,6 +63,27 @@ contract VDAStorageNodeFacet is IStorageNode {
...
+    return LibStorageNode.nodeStorage().isWithdrawlEnabled;
+  }
+
+  /**
+    * @dev see { IStorageNode }
+    */
+  function setWithdrawlEnabled(bool isEnabled) external virtual override {
```
Comment (tahpot) [inline VDA-StorageNode-Registry/contracts/facets/VDAStorageNodeFacet.sol]: `Withdrawal`

**Review by tahpot — CHANGES_REQUESTED** [review]

File: VDA-StorageNode-Registry/ChangeLog.md (hunk 2 above)
Comment (ITStar10) [reply VDA-StorageNode-Registry/ChangeLog.md]: Just then only the dates without any version number in the `ChangeLog.md`?

File: VDA-StorageNode-Registry/contracts/facets/VDAStorageNodeFacet.sol (hunk 1 above)
Comment (ITStar10) [reply VDA-StorageNode-Registry/contracts/facets/VDAStorageNodeFacet.sol]: Updated

File: VDA-StorageNode-Registry/contracts/facets/VDAStorageNodeFacet.sol (hunk 3 above)
Comment (ITStar10) [reply VDA-StorageNode-Registry/contracts/facets/VDAStorageNodeFacet.sol]: Updated

File: VDA-StorageNode-Registry/ChangeLog.md (hunk 2 above)
Comment (ITStar10) [reply VDA-StorageNode-Registry/ChangeLog.md]: Set package version as 1.0.0 in the `package.json` and `ChangeLog.md`.

**Comment by tahpot** [comment]
@ITStar10 Search your changes for `Withdrawl`, there are still 6 typos remaining.

**Review by tahpot — APPROVED** [review]

**Pull request merged**